From 3b8ecd817b636db3690575e96c9545cf6b1f116f Mon Sep 17 00:00:00 2001
From: David Wittman <dwittman@gmail.com>
Date: Tue, 1 Aug 2017 22:59:22 -0500
Subject: [PATCH] staging: lustre: declare fiemap_for_stripe static

Declare fiemap_for_stripe as static to fix sparse warnings:

> warning: symbol 'fiemap_for_stripe' was not declared. Should it be
> static?

Signed-off-by: David Wittman <dwittman@gmail.com>
---
 drivers/staging/lustre/lustre/lov/lov_object.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/lustre/lustre/lov/lov_object.c b/drivers/staging/lustre/lustre/lov/lov_object.c
index 8fc0bcc..334ecb1 100644
--- a/drivers/staging/lustre/lustre/lov/lov_object.c
+++ b/drivers/staging/lustre/lustre/lov/lov_object.c
@@ -1141,11 +1141,11 @@ struct fiemap_state {
 	bool		fs_enough;
 };
 
-int fiemap_for_stripe(const struct lu_env *env, struct cl_object *obj,
-		      struct lov_stripe_md *lsm,
-		      struct fiemap *fiemap, size_t *buflen,
-		      struct ll_fiemap_info_key *fmkey, int stripeno,
-		      struct fiemap_state *fs)
+static int fiemap_for_stripe(const struct lu_env *env, struct cl_object *obj,
+			     struct lov_stripe_md *lsm,
+			     struct fiemap *fiemap, size_t *buflen,
+			     struct ll_fiemap_info_key *fmkey, int stripeno,
+			     struct fiemap_state *fs)
 {
 	struct cl_object *subobj;
 	struct lov_obd *lov = lu2lov_dev(obj->co_lu.lo_dev)->ld_lov;
-- 
2.7.4

